### ISSUE TITLE
Insufficient contrast

### ISSUE THREAD

**Issue by maximillianh** [certbot/website#527]
We had an accessibility request from a contributor: .instructions.content .tab:not(.active) just needs color: to be black, right now it's #aaa,
